apps/web/src/hooks/useDocumentValueContext.tsx

```diff
@@ -119,16 +110,6 @@ export function DocumentValueProvider({
     { leading: false, trailing: true },
   )
 
-  const { diff } = useSyncLatteChanges({
-    content: value,
-    updateContent: setContentValue,
-    document: document,
-    commit: commit,
-    project: project,
-    devMode: devMode,
-    setDevMode: setDevMode,
-    newLatte: newLatte,
-  })
```

Comment:
use  composition as much as possible instead of prop drilling, this hook had many props, it needed none